components/dropdown/dropdown-content-mixin.js

```diff
@@ -164,6 +180,7 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 		this.noAutoClose = false;
 		this.noAutoFit = false;
 		this.noAutoFocus = false;
+		this.mobileTray = 'none';
```

Comment:
Is it possible to avoid sprouting an attribute by letting this remain `undefined` in the default case?